packages/segmented-control/src/SegmentedControl.story.tsx

```diff
@@ -76,16 +76,16 @@ Controlled.args = {
   name: 'fruit',
   children: [
     <SegmentedControlOption key="dragonfruit" value="dragonfruit">
-      Dragonfruit fruit
+      <span>Dragonfruit fruit</span>
```

Comment:
@TheSonOfThomp To make the ellipsis work I initially removed `display: flex` from the `span` element because it doesn't mesh well with `text-overflow: ellipsis;`, however `display: flex` is needed to center align the `span`, especially if there is an `Icon`. One fix was to keep flex and have consumers wrap the text in a span. This however would mean that consumers would have to updated all the children once they update to this version.

Reply:
Could we just wrap the children in another `span`?